src/index.js

```diff
@@ -26,9 +26,9 @@ function crossEnv(args) {
 
 function getCommandArgsAndEnvVars(args) {
   const envVars = getEnvVars()
-  const commandArgs = args.map(commandConvert)
-  const command = getCommand(commandArgs, envVars)
-  return [command, commandArgs, envVars]
+  const commandArgs = args.slice()
+  const command = commandConvert(getCommand(commandArgs, envVars))
+  return [commandConvert(command), commandArgs.map(commandConvert), envVars]
```

Comment:
> So the command is converted twice? Is that on purpose?

No, that was a mistake. Fixed. Good catch!

> map does not mutate its argument so no need to slice first

`getCommand` mutates its first argument (`commandArgs`), so the `.slice` is necessary.